jobs: TestJobInfoUpgradeRegressionTests failed

jobs.TestJobInfoUpgradeRegressionTests [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10466522?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10466522?buildTab=artifacts#/) on release-23.1 @ [2d33fa6b041ba0a7b362358c9b69fcabe005edba](https://github.com/cockroachdb/cockroach/commits/2d33fa6b041ba0a7b362358c9b69fcabe005edba):


```
I230609 09:00:10.325402 10896111 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r65/1:/Table/10{7-8}] 2034  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230609 09:00:10.327298 10896111 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r69/1:/Table/5{1/4-2}] 2035  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230609 09:00:10.329706 10896111 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r38/1:/Table/3{7-8}] 2036  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230609 09:00:10.332830 10896111 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r34/1:/Table/3{3-4}] 2037  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230609 09:00:10.334179 10896111 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r13/1:/Table/1{2-3}] 2038  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230609 09:00:10.335557 10896111 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r33/1:/Table/3{2-3}] 2039  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230609 09:00:10.559139 16866408 jobs/registry.go:1583  [T1,n1] 2040  AUTO CREATE STATS job 872385119179898881: stepping through state running
I230609 09:00:10.779049 16795006 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872385116792913921,upgrade=22.2-80] 2041  backfilling job_info, step0, batch0 done; resume after 872385116792913921, done false
I230609 09:00:10.865413 16795006 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872385116792913921,upgrade=22.2-80] 2042  backfilling job_info, step0, batch1 done; resume after 872385119179898881, done false
I230609 09:00:10.933474 16795006 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872385116792913921,upgrade=22.2-80] 2043  backfilling job_info, step0, batch2 done; resume after 0, done true
I230609 09:00:11.219867 16866408 jobs/registry.go:1583  [T1,n1] 2044  AUTO CREATE STATS job 872385119179898881: stepping through state succeeded
I230609 09:00:11.356493 13658185 sql/stats/automatic_stats.go:833  [T1,n1] 2045  automatically executing "CREATE STATISTICS __auto__ FROM [30] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230609 09:00:11.549773 16969174 jobs/registry.go:1583  [T1,n1] 2046  AUTO CREATE STATS job 872385122766618625: stepping through state running
I230609 09:00:11.744003 16795006 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872385116792913921,upgrade=22.2-80] 2047  backfilling job_info, step1, batch0 done; resume after 872385119179898881, done false
I230609 09:00:11.815538 16795006 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872385116792913921,upgrade=22.2-80] 2048  backfilling job_info, step1, batch0 done; resume after 872385122766618625, done false
I230609 09:00:11.859987 16795006 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872385116792913921,upgrade=22.2-80] 2049  backfilling job_info, step1, batch1 done; resume after 0, done true
I230609 09:00:11.884631 16795006 jobs/registry.go:1583  [T1,n1] 2050  MIGRATION job 872385116792913921: stepping through state succeeded
I230609 09:00:12.094775 16969174 jobs/registry.go:1583  [T1,n1] 2051  AUTO CREATE STATS job 872385122766618625: stepping through state succeeded
I230609 09:00:12.101499 10778809 jobs/wait.go:145  [T1,n1,client=127.0.0.1:33724,hostssl,user=root,migration-mgr] 2052  waited for 1 [872385116792913921] queued jobs to complete 2.395932067s
I230609 09:00:12.118206 10778809 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:33724,hostssl,user=root,migration-mgr] 2053  executing operation bump-cluster-version=22.2-80
I230609 09:00:12.123419 10778809 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:33724,hostssl,user=root,migration-mgr] 2054  executing bump-cluster-version=22.2-80 on nodes n{1}
I230609 09:00:12.124170 17039390 server/migration.go:150  [T1,n1,bump-cluster-version] 2055  active cluster version setting is now 22.2-80 (up from 22.2-79(fence))
I230609 09:00:12.198130 10778809 util/log/event_log.go:32  [T1,n1,client=127.0.0.1:33724,hostssl,user=root] 2056 ={"Timestamp":1686301164882908040,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","PlaceholderValues":["'22.2-80'"],"SettingName":"version","Value":"22.2-80"}
    job_info_storage_test.go:355: query 'SELECT count(*) FROM system.job_info WHERE job_id = 872384953098731521': expected:
        4
        
        got:
        3
        
W230609 09:00:12.206234 17047980 kv/kvserver/intentresolver/intent_resolver.go:826  [-] 2057  failed to gc transaction record: could not GC completed transaction anchored at /Table/6/1/"version"/0: node unavailable; try another peer
I230609 09:00:12.206256 10463730 sql/stats/automatic_stats.go:550  [T1,n1] 2058  quiescing auto stats refresher
I230609 09:00:12.206583 10573493 rpc/context.go:2313  [T1,n1,rnode=1,raddr=127.0.0.1:45633,class=default,rpc] 2059  connection heartbeat loop ended with err: <nil>
I230609 09:00:12.207277 14294065 jobs/registry.go:1583  [T1,n1] 2061  KEY VISUALIZER job 100: stepping through state succeeded
I230609 09:00:12.207121 10447072 server/start_listen.go:103  [T1,n1] 2060  server shutting down: instructing cmux to stop accepting
W230609 09:00:12.207385 10456826 sql/sqlliveness/slinstance/slinstance.go:334  [T1,n1] 2062  exiting heartbeat loop
W230609 09:00:12.208842 10456826 sql/sqlliveness/slinstance/slinstance.go:321  [T1,n1] 2063  exiting heartbeat loop with error: node unavailable; try another peer
E230609 09:00:12.209185 10456826 server/server_sql.go:509  [T1,n1] 2064  failed to run update of instance with new session ID: node unavailable; try another peer
E230609 09:00:12.209697 10463770 jobs/registry.go:1032  [T1,n1] 2065  error expiring job sessions: expire-sessions: failed to read query result: query execution canceled
I230609 09:00:12.210044 10503051 jobs/registry.go:1583  [T1,n1] 2066  AUTO SPAN CONFIG RECONCILIATION job 872384922406486017: stepping through state succeeded
E230609 09:00:12.210408 10503051 jobs/registry.go:988  [T1,n1] 2067  error getting live session: node unavailable; try another peer
E230609 09:00:12.211193 10463770 jobs/registry.go:1040  [T1,n1] 2068  failed to serve pause and cancel requests: txn exec: context canceled
E230609 09:00:12.213162 10463780 jobs/registry.go:1100  [T1,n1] 2069  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: failed to read query result: query execution canceled
W230609 09:00:12.213500 17047158 1@sql/event_log.go:653  [T1,n1,client=127.0.0.1:33724,hostssl,user=root] 2070  unable to save 1 entries to system.eventlog: log-event: failed to read query result: query execution canceled
W230609 09:00:12.216976 13658185 sql/stats/automatic_stats.go:819  [T1,n1] 2071  failed to create statistics on table 30: create-stats: job 872385122766618625: could not mark as succeeded: job 872385122766618625: write-job-info-insert: failed to read query result: query execution canceled
W230609 09:00:12.225290 14443267 jobs/adopt.go:518  [T1,n1] 2072  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/101,/Table/15/1/102), [txn: 7c3e4c58], [can-forward-ts] RPC error: node unavailable; try another peer
W230609 09:00:12.232215 14294065 jobs/adopt.go:518  [T1,n1] 2073  could not clear job claim: clear-job-claim: node unavailable; try another peer
I230609 09:00:12.238466 10441282 server/server_controller_orchestration.go:263  [T1,n1] 2074  server controller shutting down ungracefully
I230609 09:00:12.238605 10441282 server/server_controller_orchestration.go:274  [T1,n1] 2075  waiting for tenant servers to report stopped
W230609 09:00:12.239941 10441282 server/server_sql.go:1710  [T1,n1] 2076  server shutdown without a prior graceful drain
--- FAIL: TestJobInfoUpgradeRegressionTests (62.27s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/jobs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobInfoUpgradeRegressionTests.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-28653

jobs.TestJobInfoUpgradeRegressionTests [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10480159?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10480159?buildTab=artifacts#/) on release-23.1 @ [8cfaf3b4731273cedaa9e81600a7b1778700b702](https://github.com/cockroachdb/cockroach/commits/8cfaf3b4731273cedaa9e81600a7b1778700b702):


```
I230610 09:10:41.503470 18500917 jobs/adopt.go:261  [T1,n1,client=127.0.0.1:39414,hostssl,user=root,migration-mgr] 2077  job 872670302050811905: resuming execution
I230610 09:10:41.573060 14149003 sql/stats/automatic_stats.go:833  [T1,n1] 2078  automatically executing "CREATE STATISTICS __auto__ FROM [19] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230610 09:10:41.659375 18505109 jobs/registry.go:1583  [T1,n1] 2079  MIGRATION job 872670302050811905: stepping through state running
I230610 09:10:41.933743 18528855 jobs/registry.go:1583  [T1,n1] 2080  AUTO CREATE STATS job 872670303401738241: stepping through state running
I230610 09:10:42.699780 18528855 jobs/registry.go:1583  [T1,n1] 2081  AUTO CREATE STATS job 872670303401738241: stepping through state succeeded
I230610 09:10:42.748854 18505109 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872670302050811905,upgrade=22.2-80] 2082  backfilling job_info, step0, batch0 done; resume after 872670303401738241, done false
I230610 09:10:42.817992 18505109 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872670302050811905,upgrade=22.2-80] 2083  backfilling job_info, step0, batch0 done; resume after 0, done true
I230610 09:10:42.911078 14149003 sql/stats/automatic_stats.go:833  [T1,n1] 2084  automatically executing "CREATE STATISTICS __auto__ FROM [25] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230610 09:10:43.162959 18648771 jobs/registry.go:1583  [T1,n1] 2085  AUTO CREATE STATS job 872670307827154945: stepping through state running
I230610 09:10:43.937338 18505109 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872670302050811905,upgrade=22.2-80] 2086  backfilling job_info, step1, batch0 done; resume after 872670303401738241, done false
I230610 09:10:43.995422 18505109 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872670302050811905,upgrade=22.2-80] 2087  backfilling job_info, step1, batch1 done; resume after 872670307827154945, done false
I230610 09:10:44.053455 18505109 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872670302050811905,upgrade=22.2-80] 2088  backfilling job_info, step1, batch2 done; resume after 0, done true
I230610 09:10:44.111626 18505109 jobs/registry.go:1583  [T1,n1] 2089  MIGRATION job 872670302050811905: stepping through state succeeded
I230610 09:10:44.117354 18648771 jobs/registry.go:1583  [T1,n1] 2090  AUTO CREATE STATS job 872670307827154945: stepping through state succeeded
I230610 09:10:44.127274 11349358 2@util/log/event_log.go:32  [T1,n1] 2091 ={"Timestamp":1686388244127270219,"EventType":"runtime_stats","MemRSSBytes":526073856,"GoroutineCount":401,"MemStackSysBytes":8716288,"GoAllocBytes":214060864,"GoTotalBytes":382591352,"GoStatsStaleness":0.001467049,"HeapFragmentBytes":88158400,"HeapReservedBytes":29343744,"HeapReleasedBytes":167231488,"CGoAllocBytes":18205024,"CGoTotalBytes":46850048,"CGoCallRate":15.298818,"CPUUserPercent":193.38507,"CPUSysPercent":4.0996833,"GCPausePercent":0.80707276,"GCRunCount":194,"NetHostRecvBytes":33577,"NetHostSendBytes":33577}
I230610 09:10:44.127619 11349358 2@server/status/runtime_log.go:47  [T1,n1] 2092  runtime stats: 502 MiB RSS, 401 goroutines (stacks: 8.3 MiB), 204 MiB/365 MiB Go alloc/total(stale) (heap fragmentation: 84 MiB, heap reserved: 28 MiB, heap released: 160 MiB), 17 MiB/45 MiB CGO alloc/total (15.3 CGO/sec), 193.4/4.1 %(u/s)time, 0.8 %gc (194x), 33 KiB/33 KiB (r/w)net
I230610 09:10:44.273545 14149003 sql/stats/automatic_stats.go:833  [T1,n1] 2093  automatically executing "CREATE STATISTICS __auto__ FROM [27] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230610 09:10:44.298928 11692771 jobs/wait.go:145  [T1,n1,client=127.0.0.1:39414,hostssl,user=root,migration-mgr] 2094  waited for 1 [872670302050811905] queued jobs to complete 2.795808097s
I230610 09:10:44.299145 11692771 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:39414,hostssl,user=root,migration-mgr] 2095  executing operation bump-cluster-version=22.2-80
I230610 09:10:44.313325 11692771 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:39414,hostssl,user=root,migration-mgr] 2096  executing bump-cluster-version=22.2-80 on nodes n{1}
I230610 09:10:44.314163 18765685 server/migration.go:150  [T1,n1,bump-cluster-version] 2097  active cluster version setting is now 22.2-80 (up from 22.2-79(fence))
I230610 09:10:44.395438 11692771 util/log/event_log.go:32  [T1,n1,client=127.0.0.1:39414,hostssl,user=root] 2098 ={"Timestamp":1686388180801766322,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","PlaceholderValues":["'22.2-80'"],"SettingName":"version","Value":"22.2-80"}
    job_info_storage_test.go:355: query 'SELECT count(*) FROM system.job_info WHERE job_id = 872670085996707841': expected:
        4
        
        got:
        3
        
I230610 09:10:44.419868 11363830 sql/stats/automatic_stats.go:550  [T1,n1] 2099  quiescing auto stats refresher
W230610 09:10:44.420168 18775249 kv/kvserver/intentresolver/intent_resolver.go:826  [-] 2100  failed to gc transaction record: could not GC completed transaction anchored at /Table/6/1/"version"/0: node unavailable; try another peer
I230610 09:10:44.428131 11346087 server/start_listen.go:103  [T1,n1] 2101  server shutting down: instructing cmux to stop accepting
I230610 09:10:44.440958 15321521 jobs/registry.go:1583  [T1,n1] 2102  KEY VISUALIZER job 100: stepping through state succeeded
I230610 09:10:44.460886 11460169 rpc/context.go:2313  [T1,n1,rnode=1,raddr=127.0.0.1:45403,class=default,rpc] 2105  connection heartbeat loop ended with err: <nil>
E230610 09:10:44.441819 11363960 jobs/registry.go:1032  [T1,n1] 2103  error expiring job sessions: txn exec: context canceled
W230610 09:10:44.445514 11359693 sql/sqlliveness/slinstance/slinstance.go:334  [T1,n1] 2104  exiting heartbeat loop
I230610 09:10:44.461236 11409046 jobs/registry.go:1583  [T1,n1] 2106  AUTO SPAN CONFIG RECONCILIATION job 872670049877688321: stepping through state succeeded
E230610 09:10:44.462521 11363960 jobs/registry.go:1040  [T1,n1] 2107  failed to serve pause and cancel requests: txn exec: context canceled
W230610 09:10:44.462954 11359693 sql/sqlliveness/slinstance/slinstance.go:321  [T1,n1] 2108  exiting heartbeat loop with error: node unavailable; try another peer
E230610 09:10:44.463892 11359693 server/server_sql.go:509  [T1,n1] 2109  failed to run update of instance with new session ID: node unavailable; try another peer
E230610 09:10:44.464456 11363992 jobs/registry.go:1100  [T1,n1] 2110  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: failed to read query result: query execution canceled
W230610 09:10:44.466142 18762865 sql/create_stats.go:167  [T1,n1,intExec=create-stats] 2111  failed to cleanup StartableJob: cannot call CleanupOnRollback for a StartableJob with a non-finalized transaction
W230610 09:10:44.470609 18774198 1@sql/event_log.go:653  [T1,n1,client=127.0.0.1:39414,hostssl,user=root] 2112  unable to save 1 entries to system.eventlog: result is ambiguous: server shutdown
W230610 09:10:44.472024 14149003 sql/stats/automatic_stats.go:819  [T1,n1] 2113  failed to create statistics on table 27: create-stats: result is ambiguous: server shutdown
W230610 09:10:44.482731 11409046 jobs/adopt.go:518  [T1,n1] 2114  could not clear job claim: clear-job-claim: node unavailable; try another peer
W230610 09:10:44.483502 15321521 jobs/adopt.go:518  [T1,n1] 2115  could not clear job claim: clear-job-claim: node unavailable; try another peer
W230610 09:10:44.485039 15493674 jobs/adopt.go:518  [T1,n1] 2116  could not clear job claim: clear-job-claim: node unavailable; try another peer
I230610 09:10:44.491238 11340065 server/server_controller_orchestration.go:263  [T1,n1] 2117  server controller shutting down ungracefully
I230610 09:10:44.491412 11340065 server/server_controller_orchestration.go:274  [T1,n1] 2118  waiting for tenant servers to report stopped
W230610 09:10:44.495079 11340065 server/server_sql.go:1710  [T1,n1] 2119  server shutdown without a prior graceful drain
--- FAIL: TestJobInfoUpgradeRegressionTests (80.62s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobInfoUpgradeRegressionTests.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

jobs.TestJobInfoUpgradeRegressionTests [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10485438?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10485438?buildTab=artifacts#/) on release-23.1 @ [8cfaf3b4731273cedaa9e81600a7b1778700b702](https://github.com/cockroachdb/cockroach/commits/8cfaf3b4731273cedaa9e81600a7b1778700b702):


```
I230611 07:28:46.913768 11648969 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:57498,hostssl,user=root,migration-mgr] 2073  executing validate-cluster-version=22.2-80 on nodes n{1}
I230611 07:28:47.183430 18355706 jobs/registry.go:1583  [T1,n1] 2074  AUTO CREATE STATS job 872933377824980993: stepping through state succeeded
I230611 07:28:47.277917 11648969 upgrade/upgrademanager/manager.go:742  [T1,n1,client=127.0.0.1:57498,hostssl,user=root,migration-mgr] 2075  running Upgrade to 22.2-80: "backfill the system.jobs_info table with the payload and progress of each job in the system.jobs table"
I230611 07:28:47.278874 18498535 jobs/adopt.go:261  [T1,n1,client=127.0.0.1:57498,hostssl,user=root,migration-mgr] 2076  job 872933382294044673: resuming execution
I230611 07:28:47.372973 14058762 sql/stats/automatic_stats.go:833  [T1,n1] 2077  automatically executing "CREATE STATISTICS __auto__ FROM [6] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230611 07:28:47.422560 18501793 jobs/registry.go:1583  [T1,n1] 2078  MIGRATION job 872933382294044673: stepping through state running
I230611 07:28:47.436950 11307759 2@util/log/event_log.go:32  [T1,n1] 2079 ={"Timestamp":1686468527436945326,"EventType":"runtime_stats","MemRSSBytes":633671680,"GoroutineCount":399,"MemStackSysBytes":9175040,"GoAllocBytes":174274920,"GoTotalBytes":491302280,"GoStatsStaleness":0.002825525,"HeapFragmentBytes":141493912,"HeapReservedBytes":120635392,"HeapReleasedBytes":129040384,"CGoAllocBytes":18304864,"CGoTotalBytes":44445696,"CGoCallRate":0.20017204,"CPUUserPercent":190.96413,"CPUSysPercent":5.004301,"GCPausePercent":0.36656117,"GCRunCount":180,"NetHostRecvBytes":1826,"NetHostSendBytes":1826}
I230611 07:28:47.437209 11307759 2@server/status/runtime_log.go:47  [T1,n1] 2080  runtime stats: 604 MiB RSS, 399 goroutines (stacks: 8.8 MiB), 166 MiB/468 MiB Go alloc/total(stale) (heap fragmentation: 135 MiB, heap reserved: 115 MiB, heap released: 123 MiB), 18 MiB/42 MiB CGO alloc/total (0.2 CGO/sec), 191.0/5.0 %(u/s)time, 0.4 %gc (180x), 1.8 KiB/1.8 KiB (r/w)net
I230611 07:28:47.647598 18523739 jobs/registry.go:1583  [T1,n1] 2081  AUTO CREATE STATS job 872933383917862913: stepping through state running
I230611 07:28:48.502531 18523739 jobs/registry.go:1583  [T1,n1] 2082  AUTO CREATE STATS job 872933383917862913: stepping through state succeeded
I230611 07:28:48.648406 18501793 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872933382294044673,upgrade=22.2-80] 2083  backfilling job_info, step0, batch0 done; resume after 872933383917862913, done false
I230611 07:28:48.655114 14058762 sql/stats/automatic_stats.go:833  [T1,n1] 2084  automatically executing "CREATE STATISTICS __auto__ FROM [27] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230611 07:28:48.828396 18501793 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872933382294044673,upgrade=22.2-80] 2085  backfilling job_info, step0, batch0 done; resume after 872933388068487169, done false
I230611 07:28:48.865990 18501793 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872933382294044673,upgrade=22.2-80] 2086  backfilling job_info, step0, batch1 done; resume after 0, done true
I230611 07:28:48.879777 18627001 jobs/registry.go:1583  [T1,n1] 2087  AUTO CREATE STATS job 872933388068487169: stepping through state running
I230611 07:28:49.803134 18627001 jobs/registry.go:1583  [T1,n1] 2088  AUTO CREATE STATS job 872933388068487169: stepping through state succeeded
I230611 07:28:49.915152 18501793 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872933382294044673,upgrade=22.2-80] 2089  backfilling job_info, step1, batch0 done; resume after 872933388068487169, done false
I230611 07:28:49.983839 18501793 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=872933382294044673,upgrade=22.2-80] 2090  backfilling job_info, step1, batch0 done; resume after 0, done true
I230611 07:28:50.014051 18501793 jobs/registry.go:1583  [T1,n1] 2091  MIGRATION job 872933382294044673: stepping through state succeeded
I230611 07:28:50.056348 14058762 sql/stats/automatic_stats.go:833  [T1,n1] 2092  automatically executing "CREATE STATISTICS __auto__ FROM [51] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230611 07:28:50.158316 11648969 jobs/wait.go:145  [T1,n1,client=127.0.0.1:57498,hostssl,user=root,migration-mgr] 2093  waited for 1 [872933382294044673] queued jobs to complete 2.880064075s
I230611 07:28:50.158520 11648969 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:57498,hostssl,user=root,migration-mgr] 2094  executing operation bump-cluster-version=22.2-80
I230611 07:28:50.162924 11648969 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:57498,hostssl,user=root,migration-mgr] 2095  executing bump-cluster-version=22.2-80 on nodes n{1}
I230611 07:28:50.163617 18748731 server/migration.go:150  [T1,n1,bump-cluster-version] 2096  active cluster version setting is now 22.2-80 (up from 22.2-79(fence))
I230611 07:28:50.221989 11648969 util/log/event_log.go:32  [T1,n1,client=127.0.0.1:57498,hostssl,user=root] 2097 ={"Timestamp":1686468464840169723,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","PlaceholderValues":["'22.2-80'"],"SettingName":"version","Value":"22.2-80"}
    job_info_storage_test.go:355: query 'SELECT count(*) FROM system.job_info WHERE job_id = 872933160950038529': expected:
        4
        
        got:
        2
        
I230611 07:28:50.233755 11324836 sql/stats/automatic_stats.go:550  [T1,n1] 2098  quiescing auto stats refresher
W230611 07:28:50.233589 18754379 kv/kvserver/intentresolver/intent_resolver.go:826  [-] 2099  failed to gc transaction record: could not GC completed transaction anchored at /Table/15/1/872933392679075841/0: node unavailable; try another peer
W230611 07:28:50.234390 11320640 sql/sqlliveness/slinstance/slinstance.go:334  [T1,n1] 2100  exiting heartbeat loop
W230611 07:28:50.238611 18755239 1@sql/event_log.go:653  [T1,n1,client=127.0.0.1:57498,hostssl,user=root] 2107  unable to save 1 entries to system.eventlog: log-event: failed to read query result: query execution canceled
I230611 07:28:50.234476 11409420 rpc/context.go:2313  [T1,n1,rnode=1,raddr=127.0.0.1:41095,class=default,rpc] 2101  connection heartbeat loop ended with err: <nil>
I230611 07:28:50.235407 15342011 jobs/registry.go:1583  [T1,n1] 2102  KEY VISUALIZER job 100: stepping through state succeeded
I230611 07:28:50.236382 11304865 server/start_listen.go:103  [T1,n1] 2103  server shutting down: instructing cmux to stop accepting
I230611 07:28:50.237505 11365252 jobs/registry.go:1583  [T1,n1] 2104  AUTO SPAN CONFIG RECONCILIATION job 872933122198831105: stepping through state succeeded
E230611 07:28:50.237482 11325337 jobs/registry.go:1040  [T1,n1] 2105  failed to serve pause and cancel requests: could not query jobs table: cancel/pause-requested: failed to read query result: query execution canceled
E230611 07:28:50.238507 11325347 jobs/registry.go:1060  [T1,n1] 2106  error claiming jobs: could not query jobs table: claim-jobs: failed to read query result: query execution canceled
W230611 07:28:50.239411 11320640 sql/sqlliveness/slinstance/slinstance.go:321  [T1,n1] 2108  exiting heartbeat loop with error: node unavailable; try another peer
E230611 07:28:50.239947 11320640 server/server_sql.go:509  [T1,n1] 2109  failed to run update of instance with new session ID: node unavailable; try another peer
E230611 07:28:50.240038 15342011 jobs/registry.go:988  [T1,n1] 2110  error getting live session: node unavailable; try another peer
W230611 07:28:50.245431 15500124 jobs/adopt.go:518  [T1,n1] 2111  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/101,/Table/15/1/102), [txn: ef6bf2ee], [can-forward-ts] RPC error: node unavailable; try another peer
W230611 07:28:50.255265 14058762 sql/stats/automatic_stats.go:819  [T1,n1] 2112  failed to create statistics on table 51: create-stats: job 872933392679075841: failed to read query result: query execution canceled
I230611 07:28:50.262237 11299728 server/server_controller_orchestration.go:263  [T1,n1] 2113  server controller shutting down ungracefully
I230611 07:28:50.262317 11299728 server/server_controller_orchestration.go:274  [T1,n1] 2114  waiting for tenant servers to report stopped
W230611 07:28:50.263295 11299728 server/server_sql.go:1710  [T1,n1] 2115  server shutdown without a prior graceful drain
--- FAIL: TestJobInfoUpgradeRegressionTests (83.07s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobInfoUpgradeRegressionTests.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

jobs.TestJobInfoUpgradeRegressionTests [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10492499?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10492499?buildTab=artifacts#/) on release-23.1 @ [8cfaf3b4731273cedaa9e81600a7b1778700b702](https://github.com/cockroachdb/cockroach/commits/8cfaf3b4731273cedaa9e81600a7b1778700b702):


```
I230612 08:04:45.675087 18400633 jobs/registry.go:1583  [T1,n1] 2078  AUTO CREATE STATS job 873223568477487105: stepping through state succeeded
I230612 08:04:45.696781 11699901 upgrade/upgrademanager/manager.go:742  [T1,n1,client=127.0.0.1:51130,hostssl,user=root,migration-mgr] 2079  running Upgrade to 22.2-80: "backfill the system.jobs_info table with the payload and progress of each job in the system.jobs table"
I230612 08:04:45.700054 18474165 jobs/adopt.go:261  [T1,n1,client=127.0.0.1:51130,hostssl,user=root,migration-mgr] 2080  job 873223570363121665: resuming execution
I230612 08:04:45.721348 18475842 jobs/adopt.go:261  [T1,n1] 2081  job 873223570363121665: resuming execution
I230612 08:04:45.834945 14141362 sql/stats/automatic_stats.go:833  [T1,n1] 2082  automatically executing "CREATE STATISTICS __auto__ FROM [35] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230612 08:04:45.836555 18477847 jobs/registry.go:1583  [T1,n1] 2083  MIGRATION job 873223570363121665: stepping through state running
I230612 08:04:46.066449 18499306 jobs/registry.go:1583  [T1,n1] 2084  AUTO CREATE STATS job 873223572186923009: stepping through state running
I230612 08:04:46.152843 18518835 13@kv/kvserver/replicate_queue.go:847  [T1,n1,replicate,s1,r3/1:/System/{NodeLive…-tsd}] 2085  error processing replica: 0 of 1 live stores are able to take a new replica for the range (1 already has a voter, 0 already have a non-voter); likely not enough nodes in cluster
I230612 08:04:46.928876 18499306 jobs/registry.go:1583  [T1,n1] 2086  AUTO CREATE STATS job 873223572186923009: stepping through state succeeded
I230612 08:04:46.983180 18477847 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873223570363121665,upgrade=22.2-80] 2087  backfilling job_info, step0, batch0 done; resume after 873223572186923009, done false
I230612 08:04:47.042662 18477847 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873223570363121665,upgrade=22.2-80] 2088  backfilling job_info, step0, batch0 done; resume after 0, done true
I230612 08:04:47.212186 14141362 sql/stats/automatic_stats.go:833  [T1,n1] 2089  automatically executing "CREATE STATISTICS __auto__ FROM [45] WITH OPTIONS THROTTLING 0.9 AS OF SYSTEM TIME '-30s'"
I230612 08:04:47.452035 18630335 jobs/registry.go:1583  [T1,n1] 2090  AUTO CREATE STATS job 873223576717033473: stepping through state running
I230612 08:04:47.805981 11351635 2@util/log/event_log.go:32  [T1,n1] 2091 ={"Timestamp":1686557087805976449,"EventType":"runtime_stats","MemRSSBytes":693338112,"GoroutineCount":450,"MemStackSysBytes":9404416,"GoAllocBytes":399520240,"GoTotalBytes":559841656,"GoStatsStaleness":0.002769285,"HeapFragmentBytes":72150544,"HeapReservedBytes":35176448,"HeapReleasedBytes":12132352,"CGoAllocBytes":18224480,"CGoTotalBytes":34840576,"CGoCallRate":7.7988944,"CPUUserPercent":188.87323,"CPUSysPercent":4.6993337,"GCPausePercent":0.380437,"GCRunCount":193,"NetHostRecvBytes":82923,"NetHostSendBytes":82923}
I230612 08:04:47.806231 11351635 2@server/status/runtime_log.go:47  [T1,n1] 2092  runtime stats: 661 MiB RSS, 450 goroutines (stacks: 9.0 MiB), 381 MiB/534 MiB Go alloc/total(stale) (heap fragmentation: 69 MiB, heap reserved: 34 MiB, heap released: 12 MiB), 17 MiB/33 MiB CGO alloc/total (7.8 CGO/sec), 188.9/4.7 %(u/s)time, 0.4 %gc (193x), 81 KiB/81 KiB (r/w)net
I230612 08:04:48.168149 18477847 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873223570363121665,upgrade=22.2-80] 2093  backfilling job_info, step1, batch0 done; resume after 873223572186923009, done false
I230612 08:04:48.245925 18477847 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873223570363121665,upgrade=22.2-80] 2094  backfilling job_info, step1, batch1 done; resume after 873223576717033473, done false
I230612 08:04:48.275981 18477847 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873223570363121665,upgrade=22.2-80] 2095  backfilling job_info, step1, batch2 done; resume after 0, done true
I230612 08:04:48.303251 18477847 jobs/registry.go:1583  [T1,n1] 2096  MIGRATION job 873223570363121665: stepping through state succeeded
I230612 08:04:48.436177 11699901 jobs/wait.go:145  [T1,n1,client=127.0.0.1:51130,hostssl,user=root,migration-mgr] 2097  waited for 1 [873223570363121665] queued jobs to complete 2.736166237s
I230612 08:04:48.437328 11699901 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:51130,hostssl,user=root,migration-mgr] 2098  executing operation bump-cluster-version=22.2-80
I230612 08:04:48.443317 11699901 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:51130,hostssl,user=root,migration-mgr] 2099  executing bump-cluster-version=22.2-80 on nodes n{1}
I230612 08:04:48.444002 18725687 server/migration.go:150  [T1,n1,bump-cluster-version] 2100  active cluster version setting is now 22.2-80 (up from 22.2-79(fence))
I230612 08:04:48.538043 11699901 util/log/event_log.go:32  [T1,n1,client=127.0.0.1:51130,hostssl,user=root] 2101 ={"Timestamp":1686557024828366532,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","PlaceholderValues":["'22.2-80'"],"SettingName":"version","Value":"22.2-80"}
    job_info_storage_test.go:355: query 'SELECT count(*) FROM system.job_info WHERE job_id = 873223354939867137': expected:
        4
        
        got:
        3
        
I230612 08:04:48.571343 11350899 kv/kvserver/queue.go:614  [T1,n1,s1,r13/1:/Table/1{2-3},raft] 2102  rate limited in MaybeAdd (merge): node unavailable; try another peer
I230612 08:04:48.572129 11368803 sql/stats/automatic_stats.go:550  [T1,n1] 2103  quiescing auto stats refresher
W230612 08:04:48.572194 18735383 kv/kvserver/intentresolver/intent_resolver.go:826  [-] 2104  failed to gc transaction record: could not GC completed transaction anchored at /Table/6/1/"version"/0: node unavailable; try another peer
W230612 08:04:48.572901 11362493 sql/sqlliveness/slinstance/slinstance.go:334  [T1,n1] 2105  exiting heartbeat loop
I230612 08:04:48.573146 11452831 rpc/context.go:2313  [T1,n1,rnode=1,raddr=127.0.0.1:33287,class=default,rpc] 2106  connection heartbeat loop ended with err: <nil>
I230612 08:04:48.573436 15371954 jobs/registry.go:1583  [T1,n1] 2107  KEY VISUALIZER job 100: stepping through state succeeded
I230612 08:04:48.573971 11348107 server/start_listen.go:103  [T1,n1] 2108  server shutting down: instructing cmux to stop accepting
W230612 08:04:48.573904 11362493 sql/sqlliveness/slinstance/slinstance.go:321  [T1,n1] 2109  exiting heartbeat loop with error: node unavailable; try another peer
E230612 08:04:48.575325 11362493 server/server_sql.go:509  [T1,n1] 2110  failed to run update of instance with new session ID: node unavailable; try another peer
E230612 08:04:48.575434 15526301 jobs/registry.go:988  [T1,n1] 2111  error getting live session: node unavailable; try another peer
I230612 08:04:48.575715 11408742 jobs/registry.go:1583  [T1,n1] 2112  AUTO SPAN CONFIG RECONCILIATION job 873223317004091393: stepping through state succeeded
W230612 08:04:48.576192 18733688 1@sql/event_log.go:653  [T1,n1,client=127.0.0.1:51130,hostssl,user=root] 2113  unable to save 1 entries to system.eventlog: log-event: failed to read query result: query execution canceled
E230612 08:04:48.579992 11368859 jobs/registry.go:1032  [T1,n1] 2114  error expiring job sessions: expire-sessions: failed to read query result: query execution canceled
E230612 08:04:48.580145 11368859 jobs/registry.go:1040  [T1,n1] 2115  failed to serve pause and cancel requests: txn exec: context canceled
W230612 08:04:48.582310 14141362 sql/stats/automatic_stats.go:819  [T1,n1] 2116  failed to create statistics on table 45: create-stats: job 873223576717033473: node liveness error: restarting in background
E230612 08:04:48.584829 11368869 jobs/registry.go:1100  [T1,n1] 2117  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: failed to read query result: query execution canceled
I230612 08:04:48.588695 11341748 server/server_controller_orchestration.go:263  [T1,n1] 2118  server controller shutting down ungracefully
I230612 08:04:48.588852 11341748 server/server_controller_orchestration.go:274  [T1,n1] 2119  waiting for tenant servers to report stopped
W230612 08:04:48.590099 11341748 server/server_sql.go:1710  [T1,n1] 2120  server shutdown without a prior graceful drain
--- FAIL: TestJobInfoUpgradeRegressionTests (81.07s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobInfoUpgradeRegressionTests.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

jobs.TestJobInfoUpgradeRegressionTests [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10492502?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10492502?buildTab=artifacts#/) on release-23.1 @ [8cfaf3b4731273cedaa9e81600a7b1778700b702](https://github.com/cockroachdb/cockroach/commits/8cfaf3b4731273cedaa9e81600a7b1778700b702):


```
I230612 08:28:13.287964 38611 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1898  executing bump-cluster-version=22.2-77(fence) on nodes n{1}
I230612 08:28:13.289199 53775 server/migration.go:150  [T1,n1,bump-cluster-version] 1899  active cluster version setting is now 22.2-77(fence) (up from 22.2-76)
I230612 08:28:13.290609 38611 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1900  executing operation validate-cluster-version=22.2-78
I230612 08:28:13.292661 38611 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1901  executing validate-cluster-version=22.2-78 on nodes n{1}
I230612 08:28:13.294756 38611 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1902  executing operation bump-cluster-version=22.2-78
I230612 08:28:13.296922 38611 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1903  executing bump-cluster-version=22.2-78 on nodes n{1}
I230612 08:28:13.298388 53808 server/migration.go:150  [T1,n1,bump-cluster-version] 1904  active cluster version setting is now 22.2-78 (up from 22.2-77(fence))
I230612 08:28:13.322831 38611 upgrade/upgrademanager/manager.go:517  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1905  stepping through 22.2-80
I230612 08:28:13.323032 38611 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1906  executing operation bump-cluster-version=22.2-79(fence)
I230612 08:28:13.324748 38611 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1907  executing bump-cluster-version=22.2-79(fence) on nodes n{1}
I230612 08:28:13.325961 53846 server/migration.go:150  [T1,n1,bump-cluster-version] 1908  active cluster version setting is now 22.2-79(fence) (up from 22.2-78)
I230612 08:28:13.327705 38611 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1909  executing operation validate-cluster-version=22.2-80
I230612 08:28:13.330050 38611 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1910  executing validate-cluster-version=22.2-80 on nodes n{1}
I230612 08:28:13.439805 38611 upgrade/upgrademanager/manager.go:742  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1911  running Upgrade to 22.2-80: "backfill the system.jobs_info table with the payload and progress of each job in the system.jobs table"
I230612 08:28:13.440235 53885 jobs/adopt.go:261  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1912  job 873228184232460289: resuming execution
I230612 08:28:13.485891 53891 jobs/registry.go:1583  [T1,n1] 1913  MIGRATION job 873228184232460289: stepping through state running
I230612 08:28:13.779245 53891 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873228184232460289,upgrade=22.2-80] 1914  backfilling job_info, step0, batch0 done; resume after 873228184232460289, done false
I230612 08:28:13.803917 53891 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873228184232460289,upgrade=22.2-80] 1915  backfilling job_info, step0, batch1 done; resume after 0, done true
I230612 08:28:14.039946 53891 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873228184232460289,upgrade=22.2-80] 1916  backfilling job_info, step1, batch0 done; resume after 873228184232460289, done false
I230612 08:28:14.053794 53891 upgrade/upgrades/backfill_jobs_info_table_migration.go:72  [T1,n1,job=MIGRATION id=873228184232460289,upgrade=22.2-80] 1917  backfilling job_info, step1, batch0 done; resume after 0, done true
I230612 08:28:14.080389 53891 jobs/registry.go:1583  [T1,n1] 1918  MIGRATION job 873228184232460289: stepping through state succeeded
I230612 08:28:14.139707 38611 jobs/wait.go:145  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1919  waited for 1 [873228184232460289] queued jobs to complete 699.549807ms
I230612 08:28:14.140074 38611 upgrade/upgrademanager/manager.go:657  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1920  executing operation bump-cluster-version=22.2-80
I230612 08:28:14.142409 38611 upgrade/upgradecluster/cluster.go:121  [T1,n1,client=127.0.0.1:47176,hostssl,user=root,migration-mgr] 1921  executing bump-cluster-version=22.2-80 on nodes n{1}
I230612 08:28:14.143568 53921 server/migration.go:150  [T1,n1,bump-cluster-version] 1922  active cluster version setting is now 22.2-80 (up from 22.2-79(fence))
I230612 08:28:14.169219 38611 util/log/event_log.go:32  [T1,n1,client=127.0.0.1:47176,hostssl,user=root] 1923 ={"Timestamp":1686558470166488871,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","PlaceholderValues":["'22.2-80'"],"SettingName":"version","Value":"22.2-80"}
    job_info_storage_test.go:355: query 'SELECT count(*) FROM system.job_info WHERE job_id = 873228096470319105': expected:
        4
        
        got:
        3
        
I230612 08:28:14.177445 47546 jobs/registry.go:1583  [T1,n1] 1924  KEY VISUALIZER job 100: stepping through state succeeded
W230612 08:28:14.178088 54025 kv/kvserver/intentresolver/intent_resolver.go:614  [-] 1925  failed to cleanup transaction intents: failed to resolve intents: kv/kvserver/intentresolver/intent_resolver.go:984: stopping
I230612 08:28:14.178767 37288 server/start_listen.go:103  [T1,n1] 1926  server shutting down: instructing cmux to stop accepting
I230612 08:28:14.178893 37660 sql/stats/automatic_stats.go:550  [T1,n1] 1927  quiescing auto stats refresher
I230612 08:28:14.182917 38042 rpc/context.go:2313  [T1,n1,rnode=1,raddr=127.0.0.1:41419,class=default,rpc] 1928  connection heartbeat loop ended with err: <nil>
W230612 08:28:14.185342 37495 sql/sqlliveness/slinstance/slinstance.go:334  [T1,n1] 1930  exiting heartbeat loop
W230612 08:28:14.185188 54056 1@sql/event_log.go:653  [T1,n1,client=127.0.0.1:47176,hostssl,user=root] 1929  unable to save 1 entries to system.eventlog: log-event: failed to read query result: query execution canceled
W230612 08:28:14.185583 37495 sql/sqlliveness/slinstance/slinstance.go:321  [T1,n1] 1931  exiting heartbeat loop with error: node unavailable; try another peer
E230612 08:28:14.185799 37495 server/server_sql.go:509  [T1,n1] 1932  failed to run update of instance with new session ID: node unavailable; try another peer
E230612 08:28:14.186214 37850 jobs/registry.go:988  [T1,n1] 1933  error getting live session: node unavailable; try another peer
W230612 08:28:14.188924 47546 jobs/adopt.go:518  [T1,n1] 1934  could not clear job claim: clear-job-claim: failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/15/1/100,/Table/15/1/101), [txn: 0e0e9eb9], [can-forward-ts] RPC error: node unavailable; try another peer
W230612 08:28:14.192452 47897 jobs/adopt.go:518  [T1,n1] 1935  could not clear job claim: clear-job-claim: node unavailable; try another peer
I230612 08:28:14.316987 37436 kv/kvclient/kvcoord/transport_race.go:104  [-] 1936  transport race promotion: ran 142 iterations on up to 16861 requests
W230612 08:28:14.471151 38540 ts/db.go:208  [T1,n1,ts-poll] 1937  error writing time series data: result is ambiguous: server shutdown
I230612 08:28:14.475475 37140 server/server_controller_orchestration.go:263  [T1,n1] 1938  server controller shutting down ungracefully
I230612 08:28:14.475602 37140 server/server_controller_orchestration.go:274  [T1,n1] 1939  waiting for tenant servers to report stopped
W230612 08:28:14.480961 37140 server/server_sql.go:1710  [T1,n1] 1940  server shutdown without a prior graceful drain
--- FAIL: TestJobInfoUpgradeRegressionTests (33.85s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobInfoUpgradeRegressionTests.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/upgrade/upgrades/backfill_job_info_table_migration.go
+++ b/pkg/upgrade/upgrades/backfill_job_info_table_migration.go
@@ -49,6 +49,7 @@ func backfillJobInfoTable(
 	for step, stmt := range []string{backfillJobInfoPayloadStmt, backfillJobInfoProgressStmt} {
 		var resumeAfter int
 		for batch, done := 0, false; !done; batch++ {
+			var lastAdded int
 			if err := d.DB.KV().Txn(ctx, func(ctx context.Context, txn *kv.Txn) error {
 				last, err := d.InternalExecutor.QueryBufferedEx(
 					ctx,
@@ -63,9 +64,8 @@ func backfillJobInfoTable(
 				if err != nil {
 					return errors.Wrap(err, "failed to backfill")
 				}
-				resumeAfter = 0
 				if len(last) == 1 && len(last[0]) == 1 && last[0][0] != tree.DNull {
-					resumeAfter = int(tree.MustBeDInt(last[0][0]))
+					lastAdded = int(tree.MustBeDInt(last[0][0]))
 				} else {
 					done = true
 				}
@@ -77,6 +77,7 @@ func backfillJobInfoTable(
 			if done {
 				break
 			}
+			resumeAfter = lastAdded
 		}
 	}
 	return nil

